pkg/cli/commands/rpkg/push/command.go

```diff
@@ -254,7 +254,12 @@ func readFromDir(dir string) (map[string]string, error) {
 	}); err != nil {
 		return nil, err
 	}
-	return resources, nil
+	newRes := make(map[string]string)
+	for key, value := range resources {
+		newKey := strings.ReplaceAll(key, "\\", "/")
```

Comment:
Using something like [filepath.ToSlash](https://pkg.go.dev/path/filepath#ToSlash) seems to me a safer alternative to achieve this.

Reply:
Also, IMHO it makes the code more readable

Reply:
hi @kispaljr !

thanks for your reply! GO is far away not my primary language, so I am not aware of all the functionality. Let me fix it as you've suggested.

cheers